TreeOutputLib/OneTreeOneFile/OneTreeOneFile.py

```diff
@@ -42,48 +42,16 @@ def writeOutput(self, tree_groups, time):
                     if filename not in files_in_folder:
                         string = ""
                         string += 'time' + delimiter + 'x' + delimiter + 'y'
-                        for geometry_output in self.geometry_outputs:
-                            string += delimiter + geometry_output
-                        for parameter_output in self.parameter_outputs:
-                            string += delimiter + parameter_output
-                        for growth_output in self.growth_outputs:
-                            string += delimiter + growth_output
-                            growth_information[growth_output] = "NaN"
-                        for network_output in self.network_outputs:
-                            string += delimiter + network_output
+                        string = OneTimestepOneFile.addSelectedHeadings(
```

Comment:
As you are using inheritance you can also just do it like
```suggestion
                        string = self.addSelectedHeadings(
```

please change throughout the code

Reply:
solved